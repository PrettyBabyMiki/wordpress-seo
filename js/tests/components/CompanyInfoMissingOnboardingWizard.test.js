import { mount } from "enzyme";
import CompanyInfoMissingOnboardingWizard from "../../src/components/CompanyInfoMissingOnboardingWizard";

describe( "CompanyInfoMissingOnboardingWizard", () => {
	it( "matches the snapshot", () => {
		const tree = mount(
			<CompanyInfoMissingOnboardingWizard
				properties={ {
					message: "Testing.",
					link: "https://example.com/",
				} }
				stepState={ {
					fieldValues: {
						"publishing-entity": {
							publishingEntityCompanyName: "company",
							publishingEntityCompanyLogo: "https://example.com/logo.png",
						},
					},
				} }
			/>,
		);

		expect( tree ).toMatchSnapshot();
	} );
} );
