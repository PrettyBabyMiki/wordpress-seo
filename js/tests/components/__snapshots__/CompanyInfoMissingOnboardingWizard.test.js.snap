// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CompanyInfoMissingOnboardingWizard matches the snapshot 1`] = `
<CompanyInfoMissingOnboardingWizard
  properties={
    Object {
      "link": "https://example.com/",
      "message": "Testing.",
    }
  }
  stepState={
    Object {
      "fieldValues": Object {
        "publishing-entity": Object {
          "publishingEntityCompanyLogo": "https://example.com/logo.png",
          "publishingEntityCompanyName": "company",
        },
      },
    }
  }
/>
`;
